### ISSUE TITLE
schemaVersion is not defined in *ipmi_sensor.Ipmi

### ISSUE THREAD

**Issue by dearco** [influxdata/telegraf#4555]
I noticed this issue: https://github.com/influxdata/telegraf/issues/3416
then I install the latest master branch rpm package on my centos7, it's version
```
Telegraf unknown (git: master 3268937c)
```
but I still get the error:
```
2018/08/14 21:26:39 I! Using config file: /etc/telegraf/telegraf.conf
2018/08/14 21:26:39 E! Error parsing /etc/telegraf/telegraf.conf, line 1983: field corresponding to `schemaVersion' is not defined in `*ipmi_sensor.Ipmi'
```
thank you!

**Comment by glinton** [comment]
Good catch, it looks like there may be a typo in the example `telegraf.conf` file, can you try `metric_version` instead?

**Comment by glinton** [comment]
It looks like that's the case. Closed by 027016aea2cd65254cdf11d70413fb74dbd763a0.
